app/scenes/channel/channel_dropdown/channel_dropdown.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes} from 'react';
+import {ScrollView, Text, TouchableOpacity, View} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+function ChannelDropdown(props) {
+    const items = [{
+        title: 'View Info',
+        action: () => true
+    }, {
+        title: 'Notification Preferences',
+        action: () => true
+    }, {
+        title: 'Add Members',
+        action: () => true
+    }, {
+        title: 'View Members',
+        action: () => true
+    }, {
+        title: 'Edit Channel Header',
+        action: () => true
+    }, {
+        title: 'Edit Channel Purpose',
+        action: () => true
+    }, {
+        title: 'Rename Channel',
+        action: () => true
+    }, {
+        title: 'Leave Channel',
+        action: () => true
+    }, {
+        title: 'Add to Favorites',
+        action: () => true
+    }];
+
+    return (
+        <View style={{flex: 1}}>
+            <ScrollView>
```

Comment:
if we endup doing this as a component and passing the `items` as a prop, I would feel much more at ease if this is a `ListView` although is not going to get a very big list we should think about performance all the time